packages/rhi-webgl/src/GLRenderTarget.ts

```
@@ -271,33 +323,27 @@ export class GLRenderTarget implements IPlatformRenderTarget {
-      this._MSAAColorRenderBuffers[i] = MSAAColorRenderBuffer;
       this._blitDrawBuffers[i] = gl.NONE;
 
-      gl.bindRenderbuffer(gl.RENDERBUFFER, MSAAColorRenderBuffer);
-      gl.renderbufferStorageMultisample(
-        gl.RENDERBUFFER,
-        antiAliasing,
-        /** @ts-ignore */
-        (this._target.getColorTexture(i)._platformTexture as GLTexture)._formatDetail.internalFormat,
-        width,
-        height
+      const internalFormat = /** @ts-ignore */
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)